charts/incubator/omada-controller/questions.yaml

```
@@ -490,6 +490,13 @@ questions:
         - variable: data
           label: "App data Storage"
           description: "Stores the Application data."
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: work
+          label: "App working Storage"
+          description: "Stores the Application data between resets."
```

```suggestion
    schema:
      additional_attrs: true
      type: dict
      attrs:
        - variable: work
          label: "App working Storage"
          description: "Stores the Application data between resets."
```
` - variable: data`
until
`# Include{persistenceAdvanced}`
Is one block, copy it and paste it underneath. then make the required changes for `work` ;)